packages/sources/galaxy-test/src/endpoint/price.ts

```diff
@@ -0,0 +1,50 @@
+import { DEFAULT_WS_API_ENDPOINT } from '../config'
+import { WebSocketTransport } from '@chainlink/external-adapter-framework/transports'
+import {
+  PriceEndpoint,
+  priceEndpointInputParameters,
+} from '@chainlink/external-adapter-framework/adapter'
+import { getAccessToken } from '../util'
+import { AdapterRequestParams, PriceEndpointTypes } from '../types'
+
+// Split by the characters "_" and "/" as pairs are in this format: "markPrice_BTC/USD"
+const SPLIT_PAIR_REGEX = /[_/]+/
+
+const getPair = (params: AdapterRequestParams) => {
+  const { base = '', quote = '' } = params
```

Comment:
Are the empty strings a valid case for the params? Base and Quote should be required, so we don't need the defaults here

Reply:
Also, my understanding is that `base` is always going to be defined as it is initialized to the empty string by default.
Therefore, based on the function of logical `and` with strings, won't this function always return whatever the value for `base` is?

If I am incorrect, please clarify as I would like to know how this works.

Reply:
you're right, this was from the v2 adapter. Will update it